### PR TITLE
STOP RIGHT THERE CRIMINAL SCUM

### PR DESCRIPTION
**Pull request by Manly-Dwarf**
[Changelogs]:

:cl: KawaiiBigBoss
add: Added a new box to the SecTech containing special armor.
add: Has its own custom scream sound.
admin: Was originally an attempt at creating a new DS that didn't end well. Nothing should be damaged.
/:cl:

Nanotrasen belongs to the Nords!
Adds a box to the SecTech at Security containing a reskin of the standard officer gear, which looks similar to that of a Whiterun Guard.
The helmet applies special scream sounds.
Sprites by Not Richard but Dutch/gorram_swags. He's a cool dude!
Images:
All items equipped.
https://cdn.discordapp.com/attachments/396490982343835658/403035581908779008/unknown.png
https://i.imgur.com/aaWCDFS.png

### CODE REVIEW

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -890,7 +890,7 @@ IF YOU MODIFY THE PRODUCTS LIST OF A MACHINE, MAKE SURE TO UPDATE ITS RESUPPLY C
 	icon_deny = "sec-deny"
 	req_access_txt = "1"
 	products = list(/obj/item/restraints/handcuffs = 8, /obj/item/restraints/handcuffs/cable/zipties = 10, /obj/item/grenade/flashbang = 4, /obj/item/device/assembly/flash/handheld = 5,
-					/obj/item/reagent_containers/food/snacks/donut = 12, /obj/item/storage/box/evidence = 6, /obj/item/device/flashlight/seclite = 4, /obj/item/restraints/legcuffs/bola/energy = 7)
+					/obj/item/reagent_containers/food/snacks/donut = 12, /obj/item/storage/box/evidence = 6, /obj/item/device/flashlight/seclite = 4, /obj/item/restraints/legcuffs/bola/energy = 7, /* hippie edit */ /obj/item/storage/box/seclarp = 4 /* hippie edit end */)
```
Comment (ghost) [inline code/game/machinery/vending.dm:893]: IF YOU MODIFY THE PRODUCTS LIST OF A MACHINE, MAKE SURE TO UPDATE ITS RESUPPLY CANISTER CHARGES in vending_items.dm
https://github.com/HippieStation/HippieStation/pull/5998/files#diff-be8fc4ef536d83a7c390019dd59d14efL623

**Review by ghost — CHANGES_REQUESTED** [review]
Resupply packs won't work as intended and I have no idea if it will properly stocked on roundstart.

File: code/game/machinery/vending.dm (hunk 1 above)
Comment (Manly-Dwarf) [reply code/game/machinery/vending.dm:893]: The SecTech does not use resupply. I tested it, it spawns properly stocked on roundstart.

Comment (ghost) [reply code/game/machinery/vending.dm:893]: My bad, most issues I've had with vending machine stuff has stemmed from that.

**Comment by Manly-Dwarf** [comment]
+1
+1
+1
ok we can merge now

**Comment by Carbonhell** [comment]
1) why did you put this in the sectech vendor?if it's literally clothing, sec has a special locker containing their clothings in checkpoints and brig
2) put some pics of the sprites in the desc please

**Comment by Manly-Dwarf** [comment]
Put it in the sectech vendor because I dinnae know how to map. It's also armor, meaning that if I just put it in the "special locker" greyshirts could just bust into arrivals and get sec gear.
SecTech is the easiest and safest option.

**Comment by praisenarsie** [comment]
> put some pics of the sprites in the desc please



image

**Comment by Carbonhell** [comment]
you don't need to map, there are preset closets with a special proc, populatecontents, which i just noticed you used aswell. Just find the path to said locker, edit the proc,call ..() at the start and add your items,done

also if it is like armor,perhaps add it to the red lockers then?armor in dispensers sounds..ehh

**Comment by Manly-Dwarf** [comment]
Why have the medieval armor also crowd the red lockers? If you insist then I'll do it, but the point of adding it to the SecTech was to stop crowding, not do mapping stuff, and keep it out of hands of the greyshirts.

**Comment by Carbonhell** [comment]
normal armor vest and helmet is in the red locker,so it makes sense for a resprited item to be in the same place, but it's just minor shit,doesn't matter much, +1

File: code/modules/admin/verbs/one_click_antag.dm [hunk 2]
```diff
@@ -516,3 +516,4 @@
 /datum/admins/proc/makeRevenant()
 	new /datum/round_event/ghost_role/revenant(TRUE, TRUE)
 	return 1
+
```
Comment (Carbonhell) [inline code/modules/admin/verbs/one_click_antag.dm]: remove this?

File: hippiestation/code/modules/clothing/head/helmet.dm [hunk 3]
```diff
@@ -1,2 +1,14 @@
 /obj/item/clothing/head/helmet/knight
-	alternate_screams = list('hippiestation/sound/voice/deus_vult.ogg')
\ No newline at end of file
+	alternate_screams = list('hippiestation/sound/voice/deus_vult.ogg')
+
+/obj/item/clothing/head/helmet/larp //Skyrim, coming to BYOND in 2078!
+	name = "guard helmet"
+	desc = "A close-faced helmet. Protects the head from loose arrows and swords."
+	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDEHAIR|HIDEFACIALHAIR
+	dynamic_hair_suffix = ""
```
Comment (Carbonhell) [inline hippiestation/code/modules/clothing/head/helmet.dm:8]: are those two vars actually needed?

Comment (Manly-Dwarf) [reply hippiestation/code/modules/clothing/head/helmet.dm:8]: Yes. They properly hide the head. Without them you get a janky ass look.

**Comment by 2Dollarbill** [comment]
+1 m8, merge this shit now.

**Comment by Manly-Dwarf** [comment]
merge dis

**Comment by TrickyDickRichard** [comment]
merge            this

**Comment by earthmoonjune** [comment]
+1!!!

**Comment by Manly-Dwarf** [comment]
Hello yes it is me Maintainer Big Boss.
I can verify that the code looks fine.
You may merge this now.
:smug:

**Comment by ghost** [comment]
has a plus two and some reviews by cabroni, should be ok to merge

**Review by ghost — APPROVED** [review]

**Comment by Pyko1** [comment]
Then again didn't pass the 24 hour mark, which all non mirror, non fix PR's should be delayed.

**Comment by Pyko1** [comment]
also this should be really be a cargo pack like the crusadier set.

**Comment by Carbonhell** [comment]
@pyko1 what do you mean,the 24 hour mark is a tg thing,if a pr has the +2s and a codecheck it can be merged
Although my +1 was an head one,i didnt actually codecheck this carefully but it looks modular enough to me?

**Comment by Pyko1** [comment]
24hour

**Comment by Pyko1** [comment]
the vending machine change was not modular, only the new items.

**Comment by Manly-Dwarf** [comment]
@Pyko1 The hell do you even mean by "modular?" We don't have a mirror for vending machines on our side, you have to edit them on the /tg/ side.
If adding a single line alongside "this is a hippie edit" is not okay, then suck it up. It's the only thing you can do.

**Comment by Carbonhell** [comment]
That single line could have been avoided,pyko is right once again

**Comment by Manly-Dwarf** [comment]
And nobody told me this.
REEEEEEEEEEEEEEEEEEEEEEE

I'll put up a PR fixing the modularity soon. Next time tell me about this shizzle, CarbonHell.

**Pull request merged**